lesson8-UnitTesting/RecentlyUsedList/src/RecentlyUsedList.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace RecentlyUsedList
+{
+    public class RecentlyUsedStackSolver
```

Comment:
I'd just name it `StringStack`. Cause it's not solver, it is a stack of strings.

Reply:
done